modules/javafx.web/src/main/native/Tools/DumpRenderTree/java/DumpRenderTree.cpp

```diff
@@ -135,6 +135,26 @@ JNIEXPORT jboolean JNICALL Java_com_sun_javafx_webkit_drt_DumpRenderTree_shouldS
     return bool_to_jbool(gTestRunner->shouldStayOnPageAfterHandlingBeforeUnload());
 }
 
+JNIEXPORT jobjectArray JNICALL Java_com_sun_javafx_webkit_drt_DumpRenderTree_openPanelFiles
+    (JNIEnv* env, jclass)
+{
+    ASSERT(gTestRunner);
+    jobjectArray ret = (jobjectArray)env->NewObjectArray(gTestRunner->openPanelFiles().size(),
+                        env->FindClass("java/lang/String"), env->NewStringUTF(""));
+    int idx = 0;
+    for (auto i : gTestRunner->openPanelFiles()) {
+        env->SetObjectArrayElement(ret, idx++, env->NewStringUTF(static_cast<const char*>(i.c_str())));
```

Comment:
No need for explicit typecast as `std::string::c_str()` already returns `const char*`.